### PR TITLE
DM-17562: Broken links in afw doxygen

### PR DESCRIPTION
**Pull request by parejkoj**

### CODE REVIEW

File: doc/lsst.afw.cameraGeom/cameraGeom.rst [hunk 1]
```diff
@@ -35,6 +35,8 @@ An example of how to use the utilities to visualize a camera is available in the
 Camera Coordinate Systems
 =========================
 
+.. TODO: Convert these blocks to links to the docstrings of the constants in CameraSys (which currently lives in doxygen), once we have the ability to connect this doc to our C++ docs..
+
```
Comment (jdswinbank) [inline doc/lsst.afw.cameraGeom/cameraGeom.rst]: Where possible, please associated “TODO”s in the code with specific Jira tickets.

Comment (kfindeisen) [reply doc/lsst.afw.cameraGeom/cameraGeom.rst]: @jonathansick, is there an existing ticket for supporting links from `.rst` files to C++ components?

Comment (jonathansick) [reply doc/lsst.afw.cameraGeom/cameraGeom.rst]: Once C++ API references appear in the Sphinx-based documentation then you'll be able to use [regular Sphinx cross-reference markup](http://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing) to make those links.

I've created [DM-19574](https://jira.lsstcorp.org/browse/DM-19574) as a reference ticket that you can link a specific ticket to improve these `lsst.afw.cameraGeom` docs.

Comment (kfindeisen) [reply doc/lsst.afw.cameraGeom/cameraGeom.rst]: I've created DM-19576, which covers all of `afw` (the same issue has come up in migrating `lsst.afw.math`, and we've still got some subpackages unconverted).

**Pull request merged**